src/test/java/net/sf/jabref/importer/fileformat/BibtexParserTest.java

```diff
@@ -310,6 +310,40 @@ public void parseRecognizesEntryWhereLastFieldIsFinishedWithComma() throws IOExc
     }
 
     @Test
+    public void parseRecognizesEntryWithAtInField() throws IOException {
+        ParserResult result = BibtexParser.parse(new StringReader("@article{test,author={Ed von T@st}}"));
+
+        List<BibEntry> parsed = result.getDatabase().getEntries();
+        assertEquals(1, parsed.size());
```

Comment:
> (How that effects @tobiasdiez comment I have no clue about though.)

If `equals()` does not include the pre-entry comments, it is not sufficient in this test to state `assertEquals(expectedList, parsedList)` - because the purpose of the test is to check for the correct parsing of these comments.